rts/Lua/LuaSyncedRead.cpp

```diff
@@ -4150,6 +4151,27 @@ int LuaSyncedRead::GetUnitCosts(lua_State* L)
 	lua_pushnumber(L, unit->cost.energy);
 	return 3;
 }
+/***
+ * @function Spring.GetUnitCostTable
+ * @number unitID
+ * @treturn nil|{ metal = number, energy = number }, number buildTime
+ * @treturn number buildTime
+ */
+int LuaSyncedRead::GetUnitCostTable(lua_State* L)
+{
+	const CUnit* const unit = ParseInLosUnit(L, __func__, 1);
+	if (unit == nullptr)
+		return 0;
+	lua_createtable(L, 2, 1);
+	lua_pushnumber(L, unit->cost.metal);
+	lua_pushstring(L, "metal");
+	lua_rawset(L, -3);
+	lua_pushnumber(L, unit->cost.energy);
+	lua_pushstring(L, "energy");
+	lua_rawset(L, -3);
+	lua_pushnumber(L, unit->buildTime);
+	return 1;
```

Comment:
We return 2 values (the table with regular resources, and buildtime):
```suggestion
	return 2;
```

Reply:
Done